rebasehelper/__init__.py

```diff
@@ -22,10 +22,21 @@
 #          Nikola Forró <nforro@redhat.com>
 #          František Nečas <fifinecas@seznam.cz>
 
+import logging
 import os
 
 import pkg_resources
 
```

Comment:
Just noticed the lack of two empty lines between imports and code. Not sure if they should be here (do we consider the import below to be an import, if it's inside a `try-except` block?).

Reply:
That's a good question, we have something similat in `koji_helper.py` and there is only one blank line as well. I feel like it makes more sense to consider it as an import.

Reply:
I have the opposite opinion. After all, imports can be part of the code (python standard library does that in a lot of places), and since the block here contains also code other than imports, I consider it a code block and as such it should be separated.